src/TheBuild/Acquia/GetLatestBackupTask.php

```diff
@@ -361,10 +196,15 @@ protected function validate() {
     if (empty($this->database)) {
       $this->database = $this->site;
     }
+    // Check .env file exists.
+    if (!file_exists(".env")) {
+      throw new \BuildException(".env file is needed in the root of project with credentials see .env.example.");
```

Comment:
@acidaniel should there be an `.env.example` as part of this PR?

Reply:
@jesconstantine Yes, I just added a task to copy this file from the defaults.